### ISSUE TITLE
Root certificates not loaded on Windows

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#79044]
@chrmarti I am running 1.37 but still had to use `win-ca`.

_Originally posted by @kajwanr in https://github.com/microsoft/vscode-azure-account/issues/146#issuecomment-520469690_

**Comment by chrmarti** [comment]
@bpasero This looks like the minifier bug you hit in #74321. Was there any follow-up on that? (I will test the workaround of avoiding the `await` triggering it in tomorrow's Insiders.)

**Comment by bpasero** [comment]
@chrmarti I filed a bug against uglify-js but I think it was never fixed.

**Comment by bpasero** [comment]
There you go: https://github.com/terser-js/terser/issues/335

**Comment by gjsjohnmurray** [comment]
Just a thought. Any way this issue could lie behind the spate of reports we've had about 1.37 user not being able to install extensions unless they change what DNS server they use?

**Comment by chrmarti** [comment]
This only prevented the loading of root certificates from the OS on Windows (the Node.js baked-in ones were still in use). It should not affect any DNS related issue from what I see.

**Comment by chrmarti** [comment]
I don't see us using terser-js anymore. We seem to be using uglify-es which hasn't been updated for 1.5 years. Do we have plans for moving to something else @joao?

**Comment by chrmarti** [comment]
This appears to be fixed in the latest (1.5 years old) version of uglify-es. Not sure if there is a particular reason for staying on an older version. /fyi @jrieken

**Comment by chrmarti** [comment]
- Install OpenSSL
- Create a self-signed certificate
  - `openssl req -x509 -newkey rsa:4096 -keyout localhost_key.pem -out localhost_crt.pem -days 3650 -nodes -subj '/CN=localhost'`
- Run a server using the self-signed certificate:
```js
const https = require('https');
const fs = require('fs');

const options = {
  key: fs.readFileSync('./localhost_key.pem'),
  cert: fs.readFileSync('./localhost_crt.pem')
};

https.createServer(options, (req, res) => {
  res.writeHead(200);
  res.end('Yay!\n');
}).listen(3000, () => {
  console.log('Listening...');
});
```
- Verify a request from an extension fails due to the self-signed certificate:
```js
const https = require('https');

[...]

https.get('https://localhost:3000', res => console.log(res.statusCode))
    .on('error', console.error);
```
- `Manage computer certificates` from the WIndows start menu.
- Import `localhost_crt.pem` as trusted root certificate:
![image](https://user-images.githubusercontent.com/9205389/63919023-2e97e380-ca3e-11e9-8460-d8f493744b29.png)
- Reload the VS Code window.
- Verify the request from the extension now succeeds.
- Remove the self-signed certificate from the trusted list.
